README.md

```diff
@@ -329,6 +329,27 @@ Normally, setting required DSN information would be enough.
   ```
   - See https://docs.sentry.io/platforms/node/pluggable-integrations/ for more information
 
+### enableTracing
+- Type: `Boolean` or `Object`
+  - Default: `false`
+  - Enables the BrowserTracing integration
+  - A sampling rate can be provided if passing an object (if not provided, the default rate is 1.0):
+  ```js
+    {
+      tracesSampleRate: 0.4
+    }
+  ```
+  - Note that when enabled, the following is added to the `Vue` integration configuration:
+  ```js
+    {
+      tracing: true,
+      tracingOptions: {
+        trackComponents: true
+      }
+    }
+  ```
+  - See https://docs.sentry.io/platforms/javascript/guides/vue/#monitor-performance for more information
```

Comment:
First of all, great work. It's almost perfect. I'd just like to have it possible to be able to override the vue integration options also.

The structure of this default configuration object could look like so:

```
{
    tracesSampleRate: 1,
    vueOptions: {
        tracing: true,
        tracingOptions: {
            trackComponents: true
        }
    },
}
```
Then whatever user passes will be merged with those default options using `deepMerge`.

I'm also thinking that [server-side tracing](https://docs.sentry.io/platforms/node/performance/) should be supported. If not now then in the future.
So I'm wondering whether there should be both `enableServerTracing` and `enableClientTracing` instead of just `enableTracing`. But I can't make up my mind about that... I guess we could instead add `disableServerTracing` within this configuration object in the future, or something.

Reply:
And `BrowserTracing` integration also takes various options so it should be possible to pass those.

Reply:
Agreed that server side would be a good add, just didn't want to tackle it as part of this PR currently.